react/week1/todo-classes/src/App.css

```diff
@@ -0,0 +1,21 @@
+.App {
```

Comment:
also here please Change from "App" to "app". use same structure, lowercase in the first letter of the first word.